test/studies/shootout/meteor/kbrady/meteor-parallel.chpl

```diff
@@ -300,9 +300,9 @@ proc fillContiguousSpace(board, indx) {
 proc firstEmptyCell(cell, in min) {
   while (min == cell[0] || min == cell[1] ||
          min == cell[2] || min == cell[3] ||
-         min == cell[4]) {
+         min == cell[4]) do
```

Comment:
Oh but wait, for this specific case the block doesn't span multiple lines... I thought you were referring to cases like:

    forall ... do
      for ... {
         ... lots of lines ...
     }

which were the ones I was on the fence about.  Can you clarify?